Lib/test/test_extcall.py

```diff
@@ -221,7 +221,7 @@
     >>> f(**{1:2})
     Traceback (most recent call last):
       ...
-    TypeError: f() keywords must be strings
+    TypeError: f() keyword arguments must be strings
```

Comment:
After you have changed this I see that the former wording was more correct. I think it would be better to change the error message in `PyArg_ValidateKeywordArguments()` rather than in all other code.